### ISSUE TITLE
Use the slider as dynamic parameter

### ISSUE THREAD

**Issue by miguelo93** [jspsych/jsPsych#2648]
Hi,

I was wondering whether you could help me out with the construction of an experiment i have in mind.

I'd need to show some canvas elements (sounds and circles) with the 'time' given by participants use of a slider.

I want basically participants to use the slider as control for the time of appearance of various elements.

I don't need to collect any response in that trial.

Any suggestion on which could be a good way to do that?

Thanks!

**Comment by becky-gilbert** [comment]
Hi @miguelo93, can you say a little more about what you're trying to do? 
For instance, do participants see the slider trial(s) first, and then see the canvas element(s) after they make their slider response? Or do you want the slider to control the timing of a stimulus that is displayed simultaneously? 
Also, are the sounds and circles presented on the same trial, or do you have different trials for each stimulus type?

**Comment by miguelo93** [comment]
yes - sorry. I'd like to have the slider to control the timing of a stimulus that is displayed simultaneously?
and yes . sounds and circles are presented on the same trial.
